### PR TITLE
fixed top menu bar #532

### LINKED ISSUE #532: Top menu missing in Mobile View
Please make the top menu bar visible in mobile view e.g. by using a menu icon.

Screenshot from 2019-09-11 21-06-44

### PR DESCRIPTION
**Pull request by vinaysomawat**
Fixes #532 

Preview: http://dysfunctional-plantation.surge.sh/

Fixed Top menu bar in mobile view.
Screenshot from 2019-09-12 22-17-01

### CODE REVIEW

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please remove the code instead of commenting it out

**Comment by vinaysomawat** [comment]
@abishekvashok Done! @mariobehling

**Comment by vinaysomawat** [comment]
@mariobehling please review this pull request & merge!

File: css/custom.css [hunk 1]
```diff
@@ -169,10 +169,6 @@ nav{
   -webkit-transition: top 0.7s ease-in-out !important;
   transition: top 0.7s ease-in-out !important;
 }
-.mobile-menu-toggle{
-  position: absolute;
-  top: -7px;
```
Comment (abishekvashok) [inline css/custom.css:174]: Can you put it inside a viewpoint? Now the desktop site displays the toogle button.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: css/custom.css (hunk 1 above)
Comment (vinaysomawat) [reply css/custom.css:174]: The desktop site already displaying the toggle button. The issue is "It is not showing in mobile view". 
Do you want me to remove toggle in desktop site display?

**Comment by vinaysomawat** [comment]
@abishekvashok review it.

**Review by mariobehling — APPROVED** [review]

**Review by abishekvashok — APPROVED** [review]

**Review by liveHarshit — APPROVED** [review]

**Comment by keysangyonthan** [comment]
hey, how did you make the preview URL? I need help in making one, thanks.

**Comment by liveHarshit** [comment]
> hey, how did you make the preview URL? I need help in making one, thanks.

Learn about gh-pages.

**Comment by keysangyonthan** [comment]
ok, thanks

**Pull request merged**